src/userbase-server/server.js

```diff
@@ -286,6 +286,7 @@ async function start(express, app, userbaseConfig = {}) {
 
     // Userbase admin API
     app.use(express.static(path.join(__dirname + adminPanelDir)))
+    app.get('/access-tokens', cookieParser(), admin.authenticateAdmin, admin.getAccessTokens)
```

Comment:
Shall we put this under /admin/ too? If it's authenticated by admin, it looks like it should be there.

Reply:
Sounds good to me, that's how I first implemented it. Looked back over the spec then thought this is what you were originally proposing. In practice, it feels nice that all they'd have to do is type `/access-tokens` in the address bar (and authentication works fine since it's the same domain), but I don't have a strong opinion either way

Reply:
I think it's better under /admin/

Reply:
Nevermind. I forgot we're serving admin from root. Let's leave it as is.

Reply:
Note: followed up offline. Since we're serving the admin panel from root and the API over /v1/admin, we decided to keep it like this